example/example.js

```diff
@@ -54,6 +54,10 @@ var Example = React.createClass({
   },
   getInitialState: function() {
     return {
+      bgPrefs: {
+        bgUnits: 'mg/dL'
+        // bgUnits: 'mmol/L'
```

Comment:
Yeah, this is just the tideline example - I have a bunch of timezones commented out for easy toggling when developing too.